packages/map/src/popup/index.stories.jsx

```diff
@@ -0,0 +1,55 @@
+import { useEffect, useState } from 'react';
+import config from '@root/config';
+import { boolean, number, withKnobs } from '@storybook/addon-knobs';
+import Map from '../map';
+import data from '../../sample_data';
+import Marker from '../marker';
+import Popup from './index';
+
+
+export default { title: 'Popup', component: Popup, decorators: [withKnobs] };
+
+const Content = () => {
+  const [time, setTime] = useState(null);
+
+  useEffect(() => {
+    const timer = setInterval(() => {
+      setTime((new Date()).toLocaleString());
+    }, 1000);
+
+    return () => clearTimeout(timer);
+  });
+
+  return <strong>{time || 'Loading...'}</strong>;
+};
+
+export const Default = () => {
+  const isActive = boolean('Active', true);
```

Comment:
why do we need this?

Reply:
to check that component doesn't throw errors on unmount